### PR TITLE
HTTP Health Checker in workflow

### PR DESCRIPTION
**Pull request by cagriefegunay**
Edited `cron-data-upload.yaml` for checking the availability with valid HTTP codes before running other jobs

### CODE REVIEW

**Review by oznakn — CHANGES_REQUESTED** [review]
we need another cron workflow file for the healtcheck, and we also need a healthcheck for the file https://cdn.afetbilgi.com/latest.json

**Comment by oznakn** [comment]
cdn url yi koymak lazım

5dk ya düşürelim cron u

File: .github/workflows/cron-data-upload.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ on:
 
   schedule:
     # * is a special character in YAML so you have to quote this string
-    - cron: "*/30 * * * *" # Run every 30 minutes
+    - cron: "*/6 * * * *" # Run every 30 minutes
```
Comment (ozansz) [inline .github/workflows/cron-data-upload.yaml:8]:
```suggestion
    - cron: "*/30 * * * *" # Run every 30 minutes
```

File: .github/workflows/http-health-check.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Checking HTTP status codes are valid
+        uses: lakuapik/gh-actions-http-status@v1
+        with:
+          sites: '["https://afetbilgi.com", "https://www.afetbilgi.com", "https://maps.afetbilgi.com", "https://cdn.afetbilgi.com/latest.json", "https://pdf.afetbilgi.com"]'
```
Comment (ozansz) [inline .github/workflows/http-health-check.yaml:19]:
```suggestion
          sites: '["https://afetbilgi.com", "https://www.afetbilgi.com", "https://maps.afetbilgi.com", "https://cdn.afetbilgi.com/latest.json","https://cdn.afetbilgi.com/schema.json", "https://pdf.afetbilgi.com", "https://pdf.afetbilgi.com/index.json"]'
```

**Review by ozansz — APPROVED** [review]
nice! thans for collaboration

**Pull request merged**